src/ranged.cpp

```diff
@@ -1230,6 +1245,7 @@ static void do_aim( player &p, const item &relevant, const double min_recoil )
         // Train archery proficiencies if we are doing archery
         if( relevant.gun_skill() == skill_archery ) {
             practice_archery_proficiency( p, relevant );
+            mod_stamina_archery( p );
```

Comment:
This really needs to only be draining stamina if rapid-firing, if even then, from what I-am-Erk wrote.

Reply:
Maybe I misunderstood something; I thought rapid-firing for bows is just firing many times consecutively? Rapid-fire mode, i think, isn't a thing for bows

Reply:
Yes, that's what I meant. And if do_aim is called 100 times per turn, that means that stamina is going to be subtracted each time?

Reply:
yes, but a small value each time. It already does this for moves where it subtracts 1 move each time

Reply:
This will no longer be relevant as per I-am-Erk's suggestion so I'm marking as resolved